src/components/ConnectionsHeader/icons.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import Octicons from 'react-native-vector-icons/Octicons';
+import Entypo from 'react-native-vector-icons/Entypo';
+import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
+import AntDesign from 'react-native-vector-icons/AntDesign';
+
+export const ALARMS = {
+  'Circuit Integrity Failed': {
+    status: 'Circuit Integrity Passing',
+    icon: (status: boolean) => (
+      <Octicons
+        size={50}
+        name="circuit-board"
+        color={status ? 'red' : 'green'}
+      />
+    ),
+  },
+  'Battery in Use': {
+    status: 'Battery Disconnected',
+    icon: (status: boolean) => (
+      <Entypo size={50} name="battery" color={status ? 'red' : 'green'} />
+    ),
+  },
+  'Patient Vent Circuit Disconnected': {
+    status: 'Patient Vent Circuit Connected',
+    icon: (status: boolean) => (
+      <MaterialCommunityIcons
+        size={50}
```

Comment:
size maybe shouldn't be stored here, it should be passed from elsewhere, no? Or at the very least, have the size as a constant to be used throughout.

Reply:
Added FontSize.ts for font and Icon size